networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc

```diff
@@ -17,11 +17,14 @@ include::modules/nw-ovn-kubernetes-matrix.adoc[leveloffset=+1]
 // This is a moving target; what is included isn't valid for 4.6
 //include::modules/nw-ovn-kubernetes-metrics.adoc[leveloffset=+1]
 
+include::modules/nw-ovn-kuberentes-limitations.adoc[leveloffset=+1]
+
 .Additional resources
 
 * xref:../../networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.adoc#configuring-egress-firewall-ovn[Configuring an egress firewall for a project]
 * xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy]
 * xref:../../networking/network_policy/logging-network-policy.adoc#logging-network-policy[Logging network policy events]
 * xref:../../networking/ovn_kubernetes_network_provider/enabling-multicast.adoc#nw-ovn-kubernetes-enabling-multicast[Enabling multicast for a project]
 * xref:../../networking/ovn_kubernetes_network_provider/about-ipsec-ovn.adoc#about-ipsec-ovn[IPsec encryption configuration]
-* xref:../../rest_api/operator_apis/network-operator-openshift-io-v1.adoc#network-operator-openshift-io-v1[Network [operator.openshift.io/v1]]
+* xref:../../rest_api/operator_apis/network-operator-openshift-io-v1.adoc#network-operator-openshift-io-v1[Network [operator.openshift.io/v1]
```

Comment:
Is there a missing `]` here now? I see 2 open, 1 close now.

Reply:
I'm an idiot.  I blindly noticed that the final right bracket was not part of the anchor tag and so I removed it--thinking it was extra.  I found the Asciidoctor test for escaping a right bracket in an xref, `blah-blah\]]`.